Tests/pageObject/noticeToMarine-weekfiledownload.page.ts

```diff
@@ -239,4 +241,49 @@ const dailyfileNameData = dailyFileName[i].slice(6,14)
     expect(publicFileSizeFirst).toEqual("650 KB (.pdf)");
 
   }
+
+  public async verifySectionDoteCount()
+  {
+    const countOfDots= await this.page.$$eval('[id^="section"]' , (matches: any[]) => { return matches.map(option => option.textContent) });
+    let count=0;
+    for(let i=0;i<countOfDots.length;i++)
+   
+      if(countOfDots[i]=="---")
+      {
+        count++;
+      }
+      
+      expect(count).toEqual(3);
+
+    
+  }
+
+  public async verifyAnnualFileNameLink()
+  {
+    const fileNameLink= await this.page.$$eval("[id^='FileName'] > a" , (matches: any[]) => { return matches.map(option => option.getAttribute('href')) });
```

Comment:
never use $$Eval is the playwright recommendation

Reply:
resolved